src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Utils/SPDXExtensions.cs

```diff
@@ -31,6 +31,12 @@ public static class SPDXExtensions
     /// </summary>
     public static string GenerateSpdxId(Element element, string id) => $"SPDXRef-{element.Type}-{GetStringHash(id)}";
 
+    public static string GetSpdxId(Element element, string id)
```

Comment:
These public methods with no tests and no parameter validation are a bit concerning. I think we can make this class and all its `public` methods `internal`, but I'm not 100% sure how that will fit with #992

Reply:
I will address this in a separate PR